app/assets/javascripts/ReactComponents/table.js

```diff
@@ -207,6 +213,17 @@ var Table = React.createClass({displayName: 'Table',
         filter_type:this.props.filter_type
       });
     }
+    var footer_div = TableFooter( {columns:columns,
```

Comment:
Start the first key in an object literal on a new line (properly indented), and put spaces after colons.